appveyor.yml

```diff
@@ -28,3 +28,19 @@ after_build:
 
 build_script:
 - cmd: build-vs2017.cmd
+
+artifacts:
+- path: build\*.nupkg
+  name: core
+
+deploy:
+- provider: NuGet
+  api_key:
+    secure: Please insert key here
+  skip_symbols: false
+  artifact: core 
+  on:
+      appveyor_repo_tag: true
```

Comment:
You can control it through the settings tab of the build in appveyor and by removing `version: 0.5.{build}` from `appveyor.yml`.

Ideally we would want this to auto update itself on tag/releases. I am still toying with this idea.